### ISSUE TITLE
Prefer AVIF to WEBP for 7TV emotes

### ISSUE THREAD

**Issue by sunkhaskasis** [lay295/TwitchDownloader#426]
### Checklist

- [X] I have checked the [issue page](https://github.com/lay295/TwitchDownloader/issues) for duplicates
- [X] I am running the latest version ([download here](https://github.com/lay295/TwitchDownloader/releases/latest))

### Write your feature request here

Now that #407 is merged and the newer version of SkiaSharp is used, I believe it's safe to prefer AVIF to WEBP for 7TV.

**Comment by sunkhaskasis** [comment]
Giving it a second thought, I'm not actually sure if animated emotes would work. It's a common problem with AVIF at the moment: a lot of image editors and the like do support it, but only _static_ images.

**Comment by ScrubN** [comment]
Yeah, probably wait a few more weeks/months for AVIF to gain more traction and proper support. To be completely honest this was the first time I've heard of the format, which makes sense with how extremely new it is. Someone made a big PR to SkiaSharp about AVIF so it probably supports animated images but I'd still like to wait a bit. Not like we're losing out of much anyways, webp is already a pretty good format

**Comment by ScrubN** [comment]
Looked into this again. Seems `SKCodec` does not yet properly support AVIF despite being exposed in the `SKEncodedImageFormat` enum. :P We'll need to explicitly deny AVIF for now instead of just preferring WEBP as it causes a `null` SKCodec to be returned.
![image](https://user-images.githubusercontent.com/72096833/216795548-e505312f-690e-4ac0-8c98-caa8884d154d.png)

**Comment by ScrubN** [comment]
SkiaSharp 2.x uses Skia m88. AV1 based AVIF support was added to Skia around m100. SkiaSharp 3.0 is currently in development which will update all the way to Skia m115.

In other words, we will be able to prefer AVIF after we upgrade to SkiaSharp 3.0.

**Comment by sunkhaskasis** [comment]
@ScrubN I haven't been following this issue as of late, but have you migrated to 3.0 yet?

**Comment by ScrubN** [comment]
SkiaSharp 3.0 has not been released yet. Given the state of its development, I assume SkiaSharp 3.0 is planned for release sometime in 2024, though I wouldn't complain about being proven wrong.
